src/ontology/mondo-edit.obo

```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7886" xsd:anyURI
 
+[Term]
+id: MONDO:1030015
+name: acquired porphyria
+def: "A porphyria disorder that is acquired during the lifetime of the individual." [MONDO:patterns/acquired]
+xref: GARD:10353 {source="MONDO:GARD"}
+xref: ICD9:277.1 {source="MONDO:equivalentTo"}
+xref: MESH:D011164 {source="MONDO:relatedTo"}
+xref: NCIT:C97096 {source="MONDO:relatedTo"}
+xref: SCTID:154768004 {source="MONDO:equivalentTo"}
```

Comment:
I cannot find this x-ref in snomed. Is it the correct one? 
Also note that there are 2 SCTID which are MONDO:equivalentTo

Reply:
https://termbrowser.nhs.uk/?perspective=full&conceptId1=154768004&edition=uk-edition&release=v20240731&server=https://termbrowser.nhs.uk/sct-browser-api/snomed&langRefset=999001261000000100,999000691000001104

Reply:
changed to MONDO:mondoIsNarrowerThanSource

Reply:
this ID is inactive from what I can tell (see "details" in the link you sent).  I don't think we should use it as an x-ref.

Reply:
removed